rotkehlchen/accounting/constants.py

```diff
@@ -232,6 +233,9 @@
     )}, EventCategory.RECEIVE_GRANT: {DEFAULT: EventCategoryDetails(
         label='receive grant',
         icon='hand-coin-line',
+    )}, EventCategory.INTEREST: {DEFAULT: EventCategoryDetails(
+        label='receive interest',
+        icon='funds-line',
```

Comment:
If this is a new icon you need to do something in the frontend I am forgetting. Can you ask @rotki/frontend ?